services/paymaster/src/db/config.rs

```diff
@@ -1,32 +1,46 @@
 use crate::config_manager::config::{default_one, Config, Domain};
-use crate::constraint::TransactionVariation;
-use sqlx::{types::Json, FromRow};
+use crate::constraint::{
+    TransactionVariation, VariationOrderedInstructionConstraints, VariationProgramWhitelist,
+};
+use serde_json::Value;
+use solana_pubkey::Pubkey;
+use sqlx::{
+    types::{Json, JsonValue},
+    FromRow,
+};
 use std::collections::HashMap;
+use std::str::FromStr;
 use uuid::Uuid;
 
 use crate::db::pool;
 
-/// A row from the database that represents a domain and its transaction variations.
+/// Database row where JSON is untyped.
 #[derive(FromRow)]
-struct Row {
+struct RawRow {
     domain_id: Uuid,
     domain: String,
     enable_session_management: bool,
     enable_preflight_simulation: bool,
-    transaction_variation: Json<TransactionVariation>,
+    version: Option<String>,
```

Comment:
why are these options? we should make them nonnull in the db, no?

Reply:
https://github.com/fogo-foundation/fogo-sessions/pull/333/files/4968ade4b131d4c0ed620fb8bae8b2466787bbe7#r2561291750